.github/merge_rules.yaml

```diff
@@ -275,14 +275,16 @@
   - wanchaol
   - fduwjj
   - H-Huang
-  - aazzolini
   - kwen2501
   - XilunWu
   - wz337
   - awgu
   - fegin
-  - kumpera
-  - yhcharles
+  - kurman
+  - LucasLLC
+  - sanketpurandare
+  - shuqiangzhang
+  - tianyu-l
```

Comment:
- weifengpy

Hi Kurman, this is Wei Feng on FSDP. could I join the list?

Reply:
@weifengpy let's add your name in another PR: https://github.com/pytorch/pytorch/pull/117409
@kurman you can merge the PR if there's no conflict in rebasing. I added Wei's name in the above PR.